module-generation/scrape.py

```diff
@@ -0,0 +1,67 @@
+import os
+import json
+import requests
+from github import Github
+
+indexDir = "./scrape-data/"
+os.mkdir(indexDir)
+
+accessToken = os.environ.get('GIT_TOKEN')
+g = Github(accessToken)
+
+user = g.get_user('Terasology')
+repo = user.get_repo("Index")
+allModules = repo.get_contents("/distros/omega/gradle.properties")
+getModules = allModules.decoded_content.decode()
```

Comment:
`getModules` is a good name for a method, but not for a variable

Reply:
done